include/oneapi/dpl/internal/inclusive_scan_by_segment_impl.h

```diff
@@ -19,7 +19,7 @@
 #ifndef DPCPP_INCLUSIVE_SCAN_BY_KEY_IMPL_H_
 #define DPCPP_INCLUSIVE_SCAN_BY_KEY_IMPL_H_
 
-#include <oneapi/dpl/numeric>
+#include "../numeric"
```

Comment:
You can use `glue_numeric_impl.h` or `glue_numeric_defs.h` to avoid high-level layer